packages/contracts/src/contract-dumps.ts

```diff
@@ -211,12 +211,19 @@ export const makeStateDump = async (cfg: RollupDeployConfig): Promise<any> => {
       predeploys[name] ||
       `0xdeaddeaddeaddeaddeaddeaddeaddeaddead${i.toString(16).padStart(4, '0')}`
 
+    let def: any
```

Comment:
One problem this just raised for me is that because of `unsupported: evm` we will now be in the situation that contracts which use `kall` cannot be referenced by EVM contracts.  Solvable with interfaces but we should think about if there's something more clever to be done here.